app/lib/translate_content_purpose_fields.rb

```diff
@@ -15,7 +15,7 @@ def call
 
   def translate_fields_with_prefix(prefix)
     document_types = Array(content_store_document_types(prefix))
-    document_types += Array(subgroup_document_types(prefix) || supergroup_document_types(prefix))
+    document_types += Array(subgroup_document_types(prefix) || supergroup_document_types(prefix)) unless document_types.any?
```

Comment:
This might break some existing things.  I've raised this to fix one that I know about: https://github.com/alphagov/rummager/pull/1408

Reply:
The prepare-eu-exit ones definitely do.  The advanced-search ones (AKA the taxonomy topic finders) also do.  Apart from those, I'm pretty sure it's just our new ones.

Reply:
It looks like the advanced search finder doesn't use document type filtering (no document type filter in the content item) so I think we're in the clear there.

https://www.gov.uk/api/content/search/advanced

Did https://github.com/alphagov/rummager/pull/1408 solve this for the prepare-eu-exit ones? I saw you deployed this to prod earlier. 😄